test/test.js

```diff
@@ -63,15 +63,16 @@ describe( 'Options', () => {
 	describe( 'maxDepth', () => {
 
 		// Returns the max tree depth of the BVH
-		function getMaxDepth( node, depth = 0 ) {
+		function getMaxDepth( node ) {
 
-			if ( ! node.children ) return depth;
+			const isLeaf = 'count' in node;
```

Comment:
Perhaps we should make this a property on `MeshBVHNode`.

Reply:
You mean make `isLeaf` a property on `MeshBVHNode`? If it's only on leaf nodes it may not have a big impact on memory but I think this approach works just fine, as well.

Reply:
I meant making it a getter that returns `'count' in this`, rather than storing it.

Reply:
I've stayed away from getters and setters in the past due to performance overhead issues though maybe it won't be an issue here. I'm happy with this as-is unless you have strong feelings.

Reply:
Sure, me too.